pkg/util/objtools/abs.go

```diff
@@ -182,22 +186,25 @@ func (bkt *azureBucket) ClientSideCopy(ctx context.Context, objectName string, d
 	sourceClient := bkt.containerClient.NewBlobClient(objectName)
 	sourceClient, err := sourceClient.WithVersionID(options.SourceVersionID)
 	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("could not construct a client with the provided source version ID: %s", options.SourceVersionID))
+		return fmt.Errorf("could not construct a client with the provided source version ID: %s: %w", options.SourceVersionID, err)
 	}
 
 	response, err := sourceClient.DownloadStream(ctx, nil)
 	if err != nil {
-		return errors.Wrap(err, "failed while getting source object from Azure")
+		return fmt.Errorf("failed while getting source object from Azure: %w", err)
 	}
 	if response.ContentLength == nil {
 		return errors.New("source object from Azure did not contain a content length")
 	}
 	body := response.Body
-	if err := dstBucket.Upload(ctx, options.destinationObjectName(objectName), body, *response.ContentLength); err != nil {
+	if err = dstBucket.Upload(ctx, options.destinationObjectName(objectName), body, *response.ContentLength); err != nil {
 		_ = body.Close()
 		return errors.New("failed uploading source object from Azure to destination")
 	}
-	return errors.Wrap(body.Close(), "failed closing Azure source object reader")
+	if err = body.Close(); err != nil {
```

Comment:
[Nit] AFAICT, using `=` here instead of `:=` is unnecessary, since `err` might as well be scope local.